ArduPlane/stall_detection.cpp

```diff
@@ -0,0 +1,183 @@
+/*
+ * Copyright (C) 2020  Kraus Hamdani Aerospace Inc. All rights reserved.
+ *
+ * This file is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This file is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+ * Author: Tom Pittenger, Samuel Tabor
+ */
+
+
+#include "Plane.h"
+
+/*
+  stall detection algorithm and anti-stall management
+ */
+
+
+const AP_Param::GroupInfo Plane::StallState::var_info[] = {
+    // @Param: STALL_DETECT
```

Comment:
Pretty sure this should be
```suggestion
    // @Param: DETECT
```